src/shell/services/cloudFunctions.ts

```diff
@@ -23,10 +23,37 @@ export const cloudFunctionsApi = createApi({
         };
       },
     }),
+
+    downloadCsv: builder.query<
+      string[],
+      {
+        modelZUID: string;
+      }
+    >({
+      query: ({ modelZUID }) => {
+        return {
```

Comment:
@allenpigar the users session is not being sent with this request causing it to fail.

Reply:
user session is added using the `preparedHeaders` in utils:  https://github.com/zesty-io/manager-ui/pull/3061/files#diff-205e3a012094d2e6f027f951e9e9e609afaccb2cc2bbe29850f56c52d8e4db9dR2

i already made adjustment on the env variable in dev. seems like the env was not set properly on lower env which causes the invalid session error.